plio/views.py

```diff
@@ -465,9 +465,7 @@ def metrics(self, request, uuid):
             question_df = df[df["item_type"] == "question"].reset_index(drop=True)
             # retain only non-survey questions
             question_df = question_df[~question_df.survey].reset_index(drop=True)
-            num_non_survey_questions = len(
-                Question.objects.filter(item__plio=plio.id, survey=False)
-            )
+            non_survey_questions = len(questions.filter(survey=False))
```

Comment:
this is a number right?
```suggestion
            num_non_survey_questions = len(questions.filter(survey=False))
```